src/common/components/share/twitter.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component, PropTypes } from 'react';
+
+import styles from './styles.css';
+
+export class Tweet extends Component {
+  render() {
+    const { name } = this.props;
+    const text = encodeURIComponent(
+      `Install ${name} in seconds on Linux OSes:\nsudo snap install ${name}\n\n(Don’t have snapd? https://snapcraft.io/docs/core/install)`
```

Comment:
It is indeed awkward that the only link is not to the actual snap. But that’s pretty much unavoidable, until the snap store has a Web directory. As soon as it does, we can replace both the command line _and_ the snapd link with a directory link. The directory page can take care of people who don’t have snapd installed.